### DISCUSSION TITLE
Debug - update production files

### DISCUSSION THREAD

**Discussion by Hadkins2** [codefori/vscode-ibmi#1211] [Q&A]
I am getting the message 

`DBGP0003E  Program received unmonitored exception RNX1216: Error message CPF4203 appeared during OPEN for file SDEM100.`

This is a common problem in green screen debugging and we changed our command default to UPDPROD(*YES).  I searched google and found there was a setting in the extension settings for update production files and ticked the check box.  I have shut down debug services on the i, restarted VS Code and everything I knew to do.  But either I am not setting the extension setting correctly or it is not working.  I know that I can change my libraries to *TEST as a work around, but this has to be something simple I am overlooking.

Any help would be appreciated.

Howard Adkins

**Comment by cwbuck** [comment]
@Hadkins2 I am having this same issue. I've reported an issue #1307
